INSTALL.rst

```diff
@@ -206,3 +206,23 @@ remove a vocabulary:
 Example vocabularies are available in ``invenio_app_ils/vocabularies/data``.
 
 Vocabulary-specific configuration is available in ``config.py`` and ``invenioConfig.js``.
+
+
+CERN OAUTH DEVELOPMENT SETUP (Without docker)
```

Comment:
Yes it's temporary even though it will be useful to lay here until we finish development...in case someone needs oauth...but definitely should eventually end up to `cds-books`